department-of-veterans-affairs/va.gov-cms
# VACMS-5419: Update clone perms

## Description
Closes #5419

## Testing done
Phpunit

## QA steps
- As a content_admin, verify that you can clone node content
- As a content_creator_benefits_hubs verify that you cannot clone node content.
- As a content_creator_resources_and_support verify that you cannot clone node content.
- As a vamc_content_creator verify that you cannot clone node content.
- As a office_content_creator, verify that you cannot clone node content.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
